routes/index.js

```diff
@@ -0,0 +1,107 @@
+var express = require('express');
```

Comment:
I don't see server side rendering capabilities implemented

Reply:
@ankurkakroo2 
I need express for server side routing.